libs/agno/agno/models/litellm/chat.py

```diff
@@ -44,7 +45,10 @@ def __post_init__(self):
         if not self.api_key:
             self.api_key = getenv("LITELLM_API_KEY")
```

Comment:
In my opinion @njbrake @manuhortet we dont even need the `LITELLM_API_KEY` to be mentioned now. The reason we had it in the first place was we didn't want to set each model's key with their own `<MODEL>_API_KEY` in the init, so we unified it with just litellm's name. But it does create unnecessary confusion and with `validate_environment` it is solved.

How about we keep it simple and just do this-
```py
if not self.api_key:
         validate_environment(model=self.id, api_base=self.api_base)
```

I tested and if let's say i pass an anthropic model in the config and dont have it exported in my shell it gives proper error to me with which model is supposed to be used.

and if i have it in my env and exported. it just works. Ig that's all we need.

wdyt @manuhortet ?

Reply:
But lets first keep it backwards compatible and then we'll slowly deprecate. I'm pushing the changes

Reply:
@manuhortet nice work on making it backwards compatible also to `LITELLM_API_KEY`. This looks good, we can merge @dirkbrnd